docs/team/projectPortfolio.adoc

```diff
@@ -0,0 +1,119 @@
+= Tan Yikai - Project Portfolio
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+:imagesDir: ../images
+:stylesDir: ../stylesheets
+
+== Project: AddressBook - Level 4
```

Comment:
it shld be intelliJ...this description can copy from others' portfolio in the main repo